pkg/autoops/storage/v2/auto_ops_rule.go

```diff
@@ -179,7 +182,7 @@ func (s *autoOpsRuleStorage) ListAutoOpsRules(
 			&autoOpsRule.FeatureId,
 			&opsType,
 			&mysql.JSONObject{Val: &autoOpsRule.Clauses},
-			&autoOpsRule.TriggeredAt,
+			&tmpTriggeredAt, // ToDo: TriggeredAt is deprecated. Remove this line after migration.
```

Comment:
By removing this [line](https://github.com/bucketeer-io/bucketeer/blob/main/pkg/autoops/storage/v2/sql/auto_ops_rule/select_auto_ops_rules.sql#L6), we don't need to wait for the migration.